mesheryctl/internal/cli/root/system/update.go

```diff
@@ -27,10 +29,11 @@ var updateCmd = &cobra.Command{
 	Use:   "update",
 	Short: "Pull new Meshery images from Docker Hub.",
 	Long:  `Poll Docker Hub for new Meshery container images and pulls if new image version(s) are available.`,
```

Comment:
Can you please fix this typo ~Poll~Pull ?

Reply:
👍